common/app/model/content.scala

```diff
@@ -94,7 +94,10 @@ final case class Content(
         AmpLiveBlogSwitch.isSwitchedOn
       } else if (tags.isArticle) {
         val hasBodyBlocks: Boolean = fields.blocks.exists(b => b.body.nonEmpty)
-        AmpArticleSwitch.isSwitchedOn && hasBodyBlocks && !tags.isQuiz
+        // Some hosted pages have quiz atoms but are not tagged as quizzes
+        val hasQuizAtoms: Boolean = atoms.map(a => a.quizzes.nonEmpty).getOrElse(false)
```

Comment:
FYI @ioannakok once pointed out to me (https://github.com/guardian/mobile-apps-api/pull/2417#discussion_r1144490796) that there's a convenient method for doing this sort of thing. I _think_ this might be functionally the same?

```suggestion
        val hasQuizAtoms: Boolean = atoms.exists(a => a.quizzes.nonEmpty)
```

Reply:
Tidy! Thanks @JamieB-gu 🙏